### ISSUE TITLE
Recipes

### ISSUE THREAD

**Issue by digoIS** [digoIS/RecipeBook#3]
As a user of the application
I want to list the recipes 
So that I can see basic information about it and get detailed information.

**Comment by diego-ferreira-garcia** [comment]
Do you want to add new recipes to the list?

**Comment by digoIS** [comment]
> Do you want to add new recipes to the list?

Yes! As a user I want to add new recipes to the list. Great job!

**Comment by digoIS** [comment]
To be honest, when I click in the recipe item I want to see the recipe details so that I can to manage its information: edit it, delete it or add new ingredients. I want to see the ingredients of the recipe too.

**Comment by diego-ferreira-garcia** [comment]
So we have a manage recipe item epic here, I will break it down as follow:

As a user
I want to edit the recipe item in the list
So that I can update the recipe information

As a user
I want to delete the recipe item
So that I can remove it from the Recipe Book application

As a user
I want to add new ingredients to the recipe item
So that I can change the recipe
